exec/java-exec/src/main/java/org/apache/drill/exec/compile/ClassBuilder.java

```diff
@@ -141,7 +142,9 @@ public ClassBuilder(DrillConfig config, OptionManager optionManager) {
     // A key advantage of this method is that the code can be
     // saved and debugged, if needed.
 
-    saveCode(code, name);
+    if (cg.persistCode()) {
```

Comment:
Sounds to me that `persistCode(boolean)` is to set the persist boolean flag in code generator if it's allowed, while `persistCode()` is to check if the boolean flag is turned on or not. In such case, can we call the former as `setPersistMode(boolean)`, and call the later as `isPersistable`?